stack.go

```diff
@@ -82,7 +93,56 @@ type Stack struct {
 	UpdatedAt       time.Time     `jsonapi:"attr,updated-at,iso8601"`
 
 	// Relationships
-	Project *Project `jsonapi:"relation,project"`
+	Project                  *Project            `jsonapi:"relation,project"`
+	LatestStackConfiguration *StackConfiguration `jsonapi:"relation,latest-stack-configuration"`
+	StackDiagnostics         *StackDiagnostics   `jsonapi:"relation,stack-diagnostics"`
+}
+
+type StackConfigurationStatusTimestamps struct {
+	QueuedAt     *time.Time `jsonapi:"attr,queued-at,omitempty,rfc3339"`
+	CompletedAt  *time.Time `jsonapi:"attr,completed-at,omitempty,rfc3339"`
+	PreparingAt  *time.Time `jsonapi:"attr,preparing-at,omitempty,rfc3339"`
+	EnqueueingAt *time.Time `jsonapi:"attr,enqueueing-at,omitempty,rfc3339"`
+	CanceledAt   *time.Time `jsonapi:"attr,canceled-at,omitempty,rfc3339"`
+	ErroredAt    *time.Time `jsonapi:"attr,errored-at,omitempty,rfc3339"`
+}
```

Comment:
Why the more constrained format `rfc3339` and not `iso8601`?

Reply:
For all practical purposes, the API emits dates that fits both formats. For the purposes of parsing dates from HCP Terraform, they should be equivalent. RFC3339 is the slightly more stringent version so I usually choose it